webapp/charmhub.py

```diff
@@ -10,3 +10,333 @@ def __init__(self, d):
         "details_regex_uppercase": "[A-Za-z0-9-]*[A-Za-z][A-Za-z0-9-]*",
     }
 )
+
+
+# Dummy data for home page
+search_results = [
```

Comment:
sorry I should have thought about this in my last comment - we should probably prefix these with something like "mock_" so when real data starts coming in we can differentiate.